gensim/models/keyedvectors.py

```diff
@@ -1454,7 +1464,7 @@ def save_word2vec_format(self, fname, fvocab=None, binary=False, total_vec=None,
             if not (i == val):
                 break
             index_id_count += 1
-        keys_to_write = chain(range(0, index_id_count), store_order_vocab_keys)
+        keys_to_write = itertools.chain(range(0, index_id_count), store_order_vocab_keys)
```

Comment:
Elsewhere, I think I've followed a growing convention I've seen elsewhere to consider `itertools` as a package with a 'well known standard abbreviation' of `it`, so `import itertools as it`. I'd prefer it if gensim's project guide specifically recommended this for a small set of packages: `np`, `pd`, `it`, and maybe `pkl`, as it matches wider practices & simplifies import management. (That's especially true for numpy & itertools, where the prior practice of naming exactly which functions were used meant a jump-around just-typing burden of changing the imports every time some single function was used or removed-from-use.) But whatever project style can be unambiguously declared somewhere authoritative is OK.

Reply:
No preference from me. This one I changed merely because I saw two side-by-side imports `import itertools; from itertools import chain`, which looked weird, so I merged them into a single `import itertools`.